openshift-infra/aws_vars.yml

```diff
@@ -16,7 +16,7 @@ aws_region: us-east-2
 aws_az_1: us-east-2a
 aws_sec_group: aws-nyc-loft
 aws_key_name: rhte-apac
-tower_config: false
+tower_config: true
```

Comment:
The reason for false as the default is that lab 1 will be configuring Tower. So for our internal testing we would set this to true or use a configured AMI.